cattleshed-conf/compilers.py

```diff
@@ -2099,6 +2099,32 @@ def make_openssl(self):
             }, cv=cv))
         return compilers
 
+    def make_emacs(self):
+        emacs_vers = get_generic_versions('emacs', with_head=False)
+        compilers = []
+        for cv in emacs_vers:
+            display_name = 'Emacs'
+            version_command = ['/bin/echo', '{cv}']
+
+            compilers.append(format_value({
+                'name': 'emacs-{cv}',
+                'displayable': True,
+                'language': 'Emacs Lisp Script',
```

Comment:
The language name is too long, so probably Wandbox UI will be strange. Would you please change it into `'Emacs Lisp'`?

Reply:
Is Emacs Lisp not Lisp? A language name Lisp already exists.

Reply:
🙆 I changed to `Lisp`